tests/System.Buffers.Primitives.Tests/MemoryTests.cs

```diff
@@ -167,59 +167,6 @@ private static IEnumerable<object[]> MakePermutations(params object[][] argss)
         }
         #endregion
 
-        public static IEnumerable<object[]> ReservationPerformanceData =>
```

Comment:
If you don't think they are valuable anymore, then that is fine. I was just curious why it was removed. I thought maybe you just forgot to save the file on the other side after moving them. :)